src/css/profile/mobile/common/slider.less

```diff
@@ -82,21 +62,18 @@
 	}
 
 	&.ui-disabled {
+		opacity: 1;
 		.ui-slider-bar {
-			border-image: url(images/3_Controllers/tw_progress_bar_d.9.png) 3 fill;
-			border-image-width: 1 * @px_base;
-			opacity: 0.15;
-
+			background-color: var(--slider-bg-disabled-color);
```

Comment:
Is it correct color for the disable state?

Reply:
--slider-bg-disabled-color was defined "fade(@slider-bg-disabled-color, @slider-bg-disabled-opacity);"
 @slider-bg-disabled-color: #666666;
 @slider-bg-disabled-opacity: 15%;

The color is the value defined in the guide.

Reply:
Okay. I checked the value in base.less.